stytch/core/http/client.py

```diff
@@ -43,20 +43,45 @@ def _response_from_request(cls, r: requests.Response) -> ResponseWithJson:
             resp_json = {}
         return ResponseWithJson(response=r, json=resp_json)
 
-    def get(self, url: str, params: Optional[Dict[str, Any]]) -> ResponseWithJson:
-        resp = requests.get(url, params=params, headers=self.headers, auth=self.auth)
+    def get(
+        self,
+        url: str,
+        params: Optional[Dict[str, Any]],
+        headers: Optional[Dict[str, Any]] = None,
```

Comment:
Do we want to constrain `headers` to be `Dict[str, str]`?

Reply:
Yeah, that's a good call